### PR TITLE
Instructions for DynamoDB CDC via ClickPipes

### PR DESCRIPTION
**Pull request by kschlarman**
Closes https://github.com/ClickHouse/clickpipes-platform/issues/2569

- Rewrites and simplifies guide for setting up DynamoDB CDC with ClickPipes
- Moves guide to dedicated DynamoDB page, links to page from existing Rockset docs

### CODE REVIEW

File: docs/en/integrations/data-ingestion/dbms/dynamodb/index.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+First, you will want to enable a Kinesis stream on your DynamoDB table to capture changes in real-time. We want to do this before we create the snapshot to avoid missing any data.
+Find the AWS guide located [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/kds.html).
+
+![dynamodb-kinesis-stream.png](../images/dynamodb-kinesis-stream.png)
+
+## 2. Create the snapshot
+
+Next, we will create a snapshot of the DynamoDB table. This can be achieved through an AWS export to S3. Find the AWS guide located [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/S3DataExport.HowItWorks.html).
+You will want to do a "Full export" in the DynamoDB JSON format.
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/dbms/dynamodb/index.md:34]: Worth highlighting these requirements at least with bold text `**`

File: docs/en/integrations/data-ingestion/dbms/dynamodb/index.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+## 1. Set up Kinesis Stream
+
+First, you will want to enable a Kinesis stream on your DynamoDB table to capture changes in real-time. We want to do this before we create the snapshot to avoid missing any data.
+Find the AWS guide located [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/kds.html).
+
+![dynamodb-kinesis-stream.png](../images/dynamodb-kinesis-stream.png)
+
+## 2. Create the snapshot
+
+Next, we will create a snapshot of the DynamoDB table. This can be achieved through an AWS export to S3. Find the AWS guide located [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/S3DataExport.HowItWorks.html).
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/dbms/dynamodb/index.md:33]: btw can we leverage incremental snapshot functionality from https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/S3DataExport_Requesting.html?

File: docs/en/integrations/data-ingestion/dbms/dynamodb/index.md (hunk 1 above)
Comment (mshustov) [inline docs/en/integrations/data-ingestion/dbms/dynamodb/index.md:34]: How does S3 ClickPipe file size limitation of 1Gb play with the `full export`? https://clickhouse.com/docs/en/integrations/clickpipes/object-storage#limitations

File: docs/en/integrations/data-ingestion/dbms/dynamodb/index.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+- **Column mappings**:
+  - `ApproximateCreationDateTime`: `version`
+  - Map other fields to the appropriate destination columns as shown below
+
+![dynamodb-map-columns.png](../images/dynamodb-map-columns.png) 
+
+
+## 5. Cleanup
+
+Once the snapshot ClickPipe has finished, you can delete the snapshot table and materialized view.
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/dbms/dynamodb/index.md:130]: Is it a mandatory step?

**Review by mshustov — APPROVED** [review]

File: docs/en/integrations/data-ingestion/dbms/dynamodb/index.md (hunk 3 above)
Comment (kschlarman) [reply docs/en/integrations/data-ingestion/dbms/dynamodb/index.md:130]: Nope good point, marked as optional

File: docs/en/integrations/data-ingestion/dbms/dynamodb/index.md (hunk 1 above)
Comment (kschlarman) [reply docs/en/integrations/data-ingestion/dbms/dynamodb/index.md:34]: Sadly AWS doesn't publish this, but [a stackoverflow answer from an AWS employee](https://stackoverflow.com/questions/77539088/what-is-max-size-of-a-single-zip-object-stored-in-s3-bucket-as-a-result-of-a-dy#:~:text=DynamoDB%20optimizes%20the%20export%20procedure,files%20usually%20hover%20around%20128MB.) says:

> Although there's no specified maximum file size, individual files usually hover around 128MB.

So I think we should be good here.

File: docs/en/integrations/data-ingestion/dbms/dynamodb/index.md (hunk 2 above)
Comment (kschlarman) [reply docs/en/integrations/data-ingestion/dbms/dynamodb/index.md:33]: What use case are you imagining?

**Pull request merged**